### ISSUE TITLE
Long waiting time when parsing malware with SignedPEFile()

### ISSUE THREAD

**Issue by hxnoyd** [ralphje/signify#3]
Hi. 

I'm using signify to extract certificate metadata from malware samples, using a small python script.

I've observed that some samples cause SignedPEFile() to hang forever. When canceling the string the traceback is as follows:
```
  File "/usr/local/lib/python3.7/site-packages/signify/signed_pe.py", line 216, in signed_datas
    for certificate in self._parse_cert_table():
  File "/usr/local/lib/python3.7/site-packages/signify/signed_pe.py", line 179, in _parse_cert_table
    while position < sum(locations['certtable']):
```

I have a couple of samples that cause this problem, I can provide them if you want. (i.e. MD5:00552da691bf80b2e023381bf0566ed5839d6c86)

All samples are valid PE32 files.

Thanks in advance.

**Comment by wtfuzz** [comment]
@hxnoyd we must be working on something similar ;) I just came across this and fixed it in PR #2 yesterday.

**Comment by hxnoyd** [comment]
Thank you for the feedback. I'll give it a go!
